object_pool.go

```diff
@@ -0,0 +1,45 @@
+package astits
+
+import "sync"
+
+// poolOfTempPayload global variable is used to ease access to pool from any place of the code
+var poolOfTempPayload = &poolTempPayload{
```

Comment:
could you rename the var to `bytesPool`?

Reply:
Done